internal/federationdomain/clientregistry/clientregistry.go

```diff
@@ -165,10 +171,19 @@ func PinnipedCLI() *Client {
 			TokenEndpointAuthSigningAlgorithm: coreosoidc.RS256,
 			TokenEndpointAuthMethod:           "none",
 		},
+		IDTokenLifetimeConfiguration: 0, // never override the default timeouts for this client
 	}
 }
 
 func oidcClientCRToFositeClient(oidcClient *configv1alpha1.OIDCClient, clientSecrets []string) *Client {
+	// Allow the user to optionally override the default timeouts for these clients.
+	idTokenLifetimeOverrideInSeconds := oidcClient.Spec.TokenLifetimes.IDTokenSeconds
+	var idTokenLifetime time.Duration
+	if idTokenLifetimeOverrideInSeconds != nil {
+		// It should be safe to cast this int32 to time.Duration, because time.Duration is an int64.
+		idTokenLifetime = time.Duration(*(oidcClient.Spec.TokenLifetimes.IDTokenSeconds)) * time.Second
```

Comment:
Use `idTokenLifetimeOverrideInSeconds` instead of `(oidcClient.Spec.TokenLifetimes.IDTokenSeconds)`? Same effect, just clearer intent.

Reply:
Thanks, will do.